README.md

```diff
@@ -201,15 +210,6 @@ This will use the `lint`/`format` npm script that in generated projects uses `ts
 
 You can modify the these scripts in `package.json` to run whatever tool you prefer.
 
-
-### Formatting code
```

Comment:
Kinda piggybacking on this PR tbh... but I noticed this is outdated as the previous section covers it. Can put it on a different PR if needed.